packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -95,17 +58,7 @@ class BuySell extends Component {
                         value={filter_value}
                     />
                 </div>
-                {is_loading ? <Loading is_fullscreen={false} /> : (
-                    <ul className='buy-sell__list-wrapper'>
-                        {buy_sell_list.map(ad => (
-                            <li key={ad.id} className='buy-sell__list'>
-                                <Button primary onClick={() => this.handleClick(ad)}>
-                                    {ad.type}
-                                </Button>
-                            </li>
-                        ))}
-                    </ul>
-                )}
+                <BuySellTable table_type={filter_value} setSelectedAd={this.setSelectedAd} />
```

Comment:
should we instead turn this into a `is_type_buy`?

Reply:
changing it to table_type to be more declarative